index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Richard Portfolio Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <!-- header section -->
+    <header class="top_header">
+
+        <!-- nav menu bars -->
+        <nav class="top_nav">
+            <a href="#"><h3>Richard</h3></a>
+            <a href="#"><img src="./img/hamburger.png" alt="menu bar"></a>
+        </nav>
+    </header>
+
+    <!-- Main container   -->
+    <main class="main_container">
+        <h1 class="intro_1"> Hey There. I’m Richard.</h1>
+        <p class="intro_2"> I'm a Software Developer </p>
+        <p class="intro_3">I can help you build a product , feature or website Look through some of my work and experience! If
```

Comment:
- [ ] The headline section is part of the main section of the application however, it should be contained in a `<section>` tag because It represents a generic standalone section of the document [see this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info :+1: 
```
<main>
     <section class="headline">
     </section>
</main>
```
- [ ] Above is an example of how it should look because there would still be more sections to add to the main part of your web page :+1:

Reply:
Great observation, this has been resolved as well